app/tests/spec/lib/cropper.js

```diff
@@ -188,27 +206,35 @@ function (chai, sinon, jQuerySimulate, RouterMock, CanvasMock, p, Cropper, Ephem
       });
 
       it('resize', function () {
+        var img = { width: 100, height: 50, src: pngSrc };
+        var resized = cropper.resize(img, 0.5);
+        assert.equal(resized.src, 'data:image/jpeg');
```

Comment:
`canvas.toDataURL` should return type `data:image/png` [by default](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL) and `resize` isn't passing any arguments, so not sure why this is returning `jpeg`.